crates/dojo-world/src/manifest/mod.rs

```diff
@@ -425,7 +425,7 @@ where
             .call(
                 FunctionCall {
                     calldata: vec![],
-                    entry_point_selector: selector!("dojo_resource"),
+                    entry_point_selector: selector!("name"),
```

Comment:
we cannot use `name` here since name is used by some standards ( ercXXX )

Reply:
Oh ok, thanks for the remark 👍 
At the end, we will use `tag` which is `namespace:name`.